scripts/generate_prsn

```diff
@@ -0,0 +1,41 @@
+#!python
+from argparse import ArgumentParser
+import logging
+
+from dp.generate_prsn import logger, generate_prsn_file
+from nchelpers import CFDataset
+
+
+def main(args):
+    if args.metadata_check:
+        logger.info('Metadata Check')
+        for filepath in [args.prec, args.tasmin, args.tasmax]:
+            logger.info('')
+            logger.info('File: {}'.format(filepath))
+            try:
+                dataset = CFDataset(filepath)
+            except Exception as e:
+                logger.info('{}: {}'.format(e.__class__.__name__, e))
```

Comment:
Should this be `logger.exception`? A) then the exception info gets logged automatically, and B) this program can't continue anyways if you couldn't open one of those datasets. C) then you can drop the `else` clause, making the rest of that block a little more readable.